### PR TITLE
Add Cheerio

### LINKED ISSUE #10446: Request: Cheerio Icon
### Brand Name

Cheerio

### Website

https://cheerio.js.org/

### Popularity Metric

Fast, flexible & elegant library for parsing and manipulating HTML and XML.

Around ~6,000,000 weekly Download https://www.npmjs.com/package/cheerio
Active repository https://github.com/cheeriojs/cheerio
Upward Star History https://star-history.com/#cheeriojs/cheerio&Date

### Official Resources for Icon and Color

svg: https://github.com/cheeriojs/cheerio/blob/main/website/static/img/orange-c.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by adamrusted**
image

**Issue:** closes #10446 

**GitHub Stars:** 27,469

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/cheerio.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...59 24 0 23.1941 0 22.2V1.8C0 .8059.8059 0 1.8 0h20.4c.9941 0 1.8.8059 1.8 1.8zm-4.5418 14.9626-.4192-.3159a835.8403 835.8403 0 0 0-2.2524-1.6891c-.5658-.4233-1.133-.8477-1.7023-1.2753l-.3982-.2991-.3303.3728c-.5032.568-1.0979.8442-1.8134.8442l-.0185-.0001c-.6504-.0096-1.2016-.2635-1.6385-.755-.4164-.4679-.617-1.0721-.5646-1.7014.0526-.6335.3538-1.2013.8485-1.5995.4423-.3573 1.0467-.5242 1.6572-.4582.6104.066 1.1474.3529 1.5122.8078l.3271.4078.4182-.3137c1.2997-.9748 2.6437-1.9828 3.9674-3.0117l.184-.1431.0221-.2321c.0323-.3382-.1604-.5447-.2428-.6329-.007-.0076-.0144-.015-.0201-.0219l-.009-.0111c-.8302-1.0012-1.8558-1.7835-3.0489-2.3253a7.4076 7.4076 0 0 0-2.191-.6081c-.5812-.0714-1.3528-.1413-2.165-.0487-.8893.0981-1.7844.3565-2.6601.7679C7.623 5.13 6.5165 6.0693 5.6338 7.3104c-.5468.7591-.943 1.6163-1.1773 2.547-.2731 1.0775-.3575 1.986-.2656 2.8594.0605.5715.1509 1.2482.363 1.9323.2775.9002.7334 1.7634 1.355 2.5654.6985.9027 1.5311 1.6266 2.4753 2.1519 1.1445.6343 2.3603.9939 3.6166 1.0689a11.33 11.33 0 0 0 .6128.0185c.2542 0 .5268-.0114.7987-.0454.5321-.0619 1.0816-.1798 1.6369-.3517.6994-.2219 1.3161-.5011 1.8857-.854.9521-.5914 1.6781-1.2496 2.2194-2.0121l.3039-.428z"/></svg>
```
Comment (service-paradis) [inline icons/cheerio.svg:1]: I'm seeing some differences. Mostly in the inner `C` (yours in black):
image

Can you take another look?

**Review by service-paradis — CHANGES_REQUESTED** [review]

File: icons/cheerio.svg (hunk 1 above)
Comment (adamrusted) [reply icons/cheerio.svg:1]: Weird! I even did this one in Illustrator, so none of the Affinity Designer weirdness!

Comment (service-paradis) [reply icons/cheerio.svg:1]: We may ask another @simple-icons/icon-review member to be sure!

Comment (Mvlprem) [reply icons/cheerio.svg:1]: I don't see any differences, everything looks good to me.

Comment (adamrusted) [reply icons/cheerio.svg:1]: @Mvlprem - if you're happy with the path, could you share a screenshot of the same area @service-paradis was looking at? Feel free to review and close if those differences aren't present.

Comment (Mvlprem) [reply icons/cheerio.svg:1]: @adamrusted Maybe i wasn't paying much attention yesterday, but i can confirm there are some differences. Just to be sure I've cross checked like 5 times and ended up getting same results every single time.

Screenshot from 2024-02-22 19-36-04

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**